desktop/src/main/java/bisq/desktop/components/paymentmethods/SwiftForm.java

```diff
@@ -0,0 +1,346 @@
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.desktop.components.paymentmethods;
+
+import bisq.desktop.components.AutoTooltipButton;
+import bisq.desktop.components.InputTextField;
+import bisq.desktop.main.overlays.popups.Popup;
+import bisq.desktop.main.overlays.windows.SwiftPaymentDetails;
+import bisq.desktop.util.GUIUtil;
+import bisq.desktop.util.Layout;
+import bisq.desktop.util.validation.LengthValidator;
+
+import bisq.core.account.witness.AccountAgeWitnessService;
+import bisq.core.locale.Country;
+import bisq.core.locale.CountryUtil;
+import bisq.core.locale.Res;
+import bisq.core.payment.PaymentAccount;
+import bisq.core.payment.SwiftAccount;
+import bisq.core.payment.payload.PaymentAccountPayload;
+import bisq.core.payment.payload.SwiftAccountPayload;
+import bisq.core.trade.Trade;
+import bisq.core.util.coin.CoinFormatter;
+import bisq.core.util.validation.InputValidator;
+
+import com.jfoenix.controls.JFXTextArea;
+
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TitledPane;
+import javafx.scene.layout.GridPane;
+
+import javafx.geometry.Insets;
+
+import java.util.function.Consumer;
+
+import static bisq.common.util.Utilities.cleanString;
+import static bisq.desktop.util.FormBuilder.*;
+import static bisq.core.payment.payload.SwiftAccountPayload.*;
+
+public class SwiftForm extends PaymentMethodForm {
+    private final SwiftAccountPayload formData;
+    private CheckBox useIntermediaryCheck;
+    private LengthValidator defaultValidator = new LengthValidator(2, 34);
+    private LengthValidator swiftValidator = new LengthValidator(11, 11);
+    private LengthValidator accountNrValidator = new LengthValidator(2, 40);
+    private LengthValidator addressValidator = new LengthValidator(1, 100);
+
+    public SwiftForm(PaymentAccount paymentAccount,
+                     AccountAgeWitnessService accountAgeWitnessService,
+                     InputValidator defaultValidator, GridPane gridPane, int gridRow, CoinFormatter formatter) {
+        super(paymentAccount, accountAgeWitnessService, defaultValidator, gridPane, gridRow, formatter);
+        this.formData = ((SwiftAccount) paymentAccount).getPayload();
+        this.useIntermediaryCheck = new CheckBox(Res.get("payment.swift.use.intermediary"));
```

Comment:
```suggestion
        this.useIntermediaryCheck = new AutoTooltipCheckBox(Res.get("payment.swift.use.intermediary"));
```
Should be 
![Bildschirmfoto 2021-09-09 um 10 21 18](https://user-images.githubusercontent.com/170962/132650764-d21100a0-57cf-46a9-8410-5f404dfea48c.png)
Instead of 
![Bildschirmfoto 2021-09-09 um 10 19 33](https://user-images.githubusercontent.com/170962/132650795-5855bd73-ea2a-4e76-85c2-9358e0e123bb.png)